### PR TITLE
Details accordion

### PR DESCRIPTION
**Pull request by ianatesfa**

### CODE REVIEW

File: src/SFA.DAS.FindEmploymentSchemes.Contentful/GdsHtmlRenderers/GdsEmbeddedYoutubeContentRenderer.cs [hunk 1]
```diff
@@ -0,0 +1,76 @@
+﻿using Contentful.Core.Models;
+using System.Text;
+using System.Threading.Tasks;
+
+namespace SFA.DAS.FindEmploymentSchemes.Contentful.GdsHtmlRenderers
+{
+    /// <summary>
+    /// A renderer for an embedded YouTube video
+    /// </summary>
+    public class GdsEmbeddedYoutubeContentRenderer : IContentRenderer
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.FindEmploymentSchemes.Contentful/GdsHtmlRenderers/GdsEmbeddedYoutubeContentRenderer.cs:10]: Do these renderers not have unit tests?

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml [hunk 2]
```diff
@@ -25,20 +25,39 @@
             <h2 id="scheme-details-h2-responsibility" class="govuk-heading-m"> Employer responsibilities </h2>
             @Model.Scheme.Responsibility
             <h2 id="scheme-details-h2-benefits" class="govuk-heading-m">Benefits of @Model.Scheme.Name</h2>
-            @Model.Scheme.Benefits
-            @if (@Model.Scheme.CaseStudies != null)
+                @Model.Scheme.Benefits
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml:28]: Very minor but looks like the indentation of this line has been knocked out.

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml [hunk 3]
```diff
@@ -25,20 +25,39 @@
...
                 @Model.Scheme.Offer
-            </section>
-        }
-    </div>
-    <div id="scheme-details-inner-one-third" class="govuk-grid-column-one-third">
-        <h3 id="scheme-details-inner-h3" class="govuk-heading-m">Other schemes</h3>
+            </section>        }
+	</div>
+	<div id="scheme-details-inner-one-third" class="govuk-grid-column-one-third">
+		<h3 id="scheme-details-inner-h3" class="govuk-heading-m">Other schemes</h3>
```
Comment (MichaelYoung1981) [inline src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml:60]: Indentation looks out.

**Review by MichaelYoung1981 — CHANGES_REQUESTED** [review]

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml (hunk 2 above)
Comment (ianatesfa) [reply src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml:28]: Fixed in latest commit

File: src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml (hunk 3 above)
Comment (ianatesfa) [reply src/SFA.DAS.FindEmploymentSchemes.Web/Views/Schemes/Details.cshtml:62]: Fixed in latest commit

File: src/SFA.DAS.FindEmploymentSchemes.Contentful/GdsHtmlRenderers/GdsEmbeddedYoutubeContentRenderer.cs (hunk 1 above)
Comment (lazcool) [reply src/SFA.DAS.FindEmploymentSchemes.Contentful/GdsHtmlRenderers/GdsEmbeddedYoutubeContentRenderer.cs:10]: The renderers have only been used up to now, as part of an offline utility, run to generate C# code.

We'll definitely need unit tests for them as part of the Contenful integration (when they will run online), which is currently being developed.

As part of that development, we've updated the renderers to support the latest version of the Contentful .Net library, which has changed their signatures, so it makes sense for the unit tests to be written as part of that story.

**Review by lazcool — APPROVED** [review]

**Pull request merged**